src/components/CardForDocSIG/listForCard/List.vue

```diff
@@ -11,7 +11,7 @@
 </template>
 
 <script setup lang="ts">
-import GetCertifications from "../../../graphql/certification/Certification.gql";
+import GetCertifications from "../../../graphql/certification/queries.gql";
```

Comment:
Geralmente é usado o prefixo `Get` quando buscamos por 1 registro apenas, quando for mais de 1 registro usamos o prefixo `Load`

Reply:
Resolvido